Wyniki testów: 

FAIL: test_nowe_zadanie2 (__main__.TestZadanie.test_nowe_zadanie2)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\BSkla\OneDrive\Pulpit\Studia\Podstawy Python\Ćwiczenia\Projekt\test_Zadanie.py", line 20, in test_nowe_zadanie2
    self.assertEqual(zadanie1.status, 'nierozpoczęte')
AssertionError: 'rozpoczęte' != 'nierozpoczęte'
- rozpoczęte
+ nierozpoczęte
? +++


----------------------------------------------------------------------
Ran 2 tests in 0.001s

FAILED (failures=1)

I test sprawdza czy jeśli nie zostanie podana data rozpoczęcia to data rozpoczęcia zostanie ustawiona automatycznie na datę dzisiejszą. 

II test sprawdza czy po ustawieniu daty rozpoczęcia na datę późniejszą niż dzisiejsza to status zostanie ustawiony automatycznie na 'nierozpoczęte'. 